spec/syntax.md

```diff
@@ -272,8 +321,18 @@ declaration = let s variable [s] "=" [s] expression
 
 ### Selectors
 
-A `match` statement contains one or more **_selectors_**
-which will be used to choose one of the _variants_ during formatting.
+**_Selectors_** are a _match statement_ followed by one or more _variants_.
+_Selectors_ provide the ability for a _message_ to use a _pattern_
+that varies in content or form depending on values determined at runtime.
```

Comment:
```suggestion
A _message_ can vary its content or form
depending on values determined at runtime.
_Selectors_ make this variation possible.
```

It's not really the _pattern_ that varies (the static, syntactic construct), but the formatted output (the runtime result). Each syntactic pattern is fixed; the choice of pattern varies.